uml/sequence_diagrams/DisplayViewSequence.txt

```diff
@@ -0,0 +1,104 @@
+@startuml
+participant DeactivatedClock 
+participant DisplayViews 
+participant ClockView 
+participant StepCountView 
+participant HeartRateView 
+participant ECGView 
+participant SP02View
+participant MapView
+participant SettingsView 
+participant StepCounter
+participant HeartRateMonitor
+participant ECGMonitor
+participant SP02Monitor
+participant HeartRateMonitor
+participant DisplayError 
+
+
+opt Clock
+  DisplayViews --> ClockView : GoTo Clock

```

Comment:
I think that all of the optional transitions to and from DisplayViews shows the same info on State Diagram, and is probably clearer in that context.

Perhaps the other parts (e.g. AskStepCount, and the optional warning) could each have their own small sequence diagram.

Reply:
I think is ok if we describe in different ways, that's the purpose of the project. 

We just need to check how it comes on a word docs, maybe too big. We can check this later.